openpype/hosts/tvpaint/api/lib.py

```diff
@@ -540,3 +539,16 @@ def get_scene_bg_color(communicator=None):
     if not data:
         return None
     return data.split(" ")
+
+
+def get_layer_opacity(layer_id):
+    """Return the opacity set on layer.
+    layer_id(int): id of the layer to get the opacity
+
+    Returns:
+        str: Layer Opacity (0-100).
```

Comment:
In my opinion the function should return an int not a string:
```
int: Layer Opacity [0-100].
```

Reply:
yes !